pipelines/validate_run.py

```
@@ -811,61 +855,123 @@ def observable_text(value):
-                "non-sensitive",
-                "owned repository",
-                "public data",
-                "public documentation",
-                "read-only",
-                "sanitized input",
-                "test fixture",
-                "user-owned",
-            )
+        benign_secret_topic = (
+            r"(?![\s-]+(?:docs?|documentation|example|format|guide|help|policy|"
+            r"reset|rotation|rules?|template))"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "benign_secret_topic"

(S105)